services/static-badge/query-string-static.tester.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('Shields colorscheme color')
+  .get(
+    '/static/v1.svg?label=label&message=message&color=blue&style=_shields_test'
+  )
+  .expectJSON({ name: 'label', value: 'message', color: 'blue' })
+
+t.create('CSS named color')
+  .get(
+    '/static/v1.svg?label=label&message=message&color=whitesmoke&style=_shields_test'
+  )
+  .expectJSON({ name: 'label', value: 'message', color: 'whitesmoke' })
+
+t.create('RGB color')
+  .get(
+    '/static/v1.svg?label=label&message=message&color=rgb(123,123,123)&style=_shields_test'
+  )
+  .expectJSON({ name: 'label', value: 'message', color: 'rgb(123,123,123)' })
+
+t.create('All one color')
+  .get(
+    '/static/v1.svg?label=&messageall%20one%20color&color=red&style=_shields_test'
+  )
+  .expectJSON({ name: '', value: 'all one color', color: 'red' })
+
+t.create('Not a valid color')
+  .get(
+    '/static/v1.svg?label=label&message=message&color=notacolor&style=_shields_test'
+  )
+  .expectJSON({ name: 'label', value: 'message', color: null })
```

Comment:
These seem like they are testing functionality in `coalesceBadge` + BaseService. There's already a "set color" test at the bottom, so how about removing all of these?

If it seems important to add more tests of that code I would prefer we find a way to do it with a unit test instead of a service test, as the unit tests get run more frequently.

Reply:
Fair point.